test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -818,6 +799,8 @@ def __capture_trace_from_core(self, core_file_name, exec_path, out_path):
         gdb.execute("quit")
         quit()
         """ % (core_file_name)
+        # Remove tabs from triple quoted strings
+        gdb_python = gdb_python.replace("\t", "")
```

Comment:
If it's a triple quoted string, you have control.  If you don't want the tabs, why put them in the string in the first place?